UpdateTranslations.ps1

```diff
@@ -0,0 +1,23 @@
+Push-Location (Split-Path -Path $MyInvocation.MyCommand.Definition -Parent)
+
+$api_token = "😉";
```

Comment:
lol

Reply:
You can get your API token in POEditor Account Settings. Didn't want to share mine for obvious reasons 😄